stats/internal/otel/prometheus/config.go

```diff
@@ -101,6 +104,23 @@ func WithoutUnits() Option {
 	})
 }
 
+// WithNamespace configures the Exporter to prefix metric with the given namespace.
+// Metadata metrics such as target_info and otel_scope_info are not prefixed since these
+// have special behavior based on their name.
+func WithNamespace(ns string) Option {
```

Comment:
**NOTE**: I added this even though we are not using it just to avoid having our prometheus exporter drift too much from the original.